docs/resources-overview.md

```diff
@@ -41,6 +41,9 @@ the `knative-serving` namespace. For example, if you create a Knative Serivce in
 namespace 'foo' the corresponding Istio resources will also be in namespace
 'foo'.
 
+All of these components are run as a non-root user (uid: 1337) and disallow
```

Comment:
Shouldn't this uid be 65532? We don't set a different uid in security context and non-root user in the base image looks to be 65532.

```suggestion
All of these components are run as a non-root user (uid: 65532) and disallow
```

Reply:
Correct, thanks